app/views/devise/registrations/edit.html.erb

```diff
@@ -1,45 +1,78 @@
-<div class="container">
-  <h2>Edit <%= resource_name.to_s.humanize %></h2>
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
 
-  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-    <%= render "devise/shared/error_messages", resource: resource %>
+<div class="card">
+  <div class="card-body">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
 
-    <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-      <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
-    <% end %>
-
-    <div class="field">
-      <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br/>
-      <%= f.password_field :password, autocomplete: "new-password" %>
-      <% if @minimum_password_length %>
-        <br/>
-        <em><%= @minimum_password_length %> characters minimum</em>
+      <div class="card">
+        <h5 class="card-header">Change email</h5>
+        <div class="card-body">
+          <div class="field">
+            <%= f.label :email %><br />
+            <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control" %>
+          </div><br/>
```

Comment:
We talked about the fact that now it might be better not to change your email and name. Otherwise, we should update Stripe. Please also remove the parameters to be updated in the controller `:first_name, :last_name` in the `update_allowed_parameters` method. We can change that in the future. 👍